apps/prairielearn/src/server.ts

```diff
@@ -1733,6 +1733,10 @@ export async function initExpress(): Promise<Express> {
     '/pl/public/course_instance/:course_instance_id(\\d+)/assessment/:assessment_id(\\d+)/questions',
     (await import('./pages/publicAssessmentQuestions/publicAssessmentQuestions.js')).default,
   );
+  app.use(
+    '/pl/public/course_instance/:course_instance_id(\\d+)/assessments',
+    (await import('./pages/publicAssessments/publicAssessments.js')).default,
+  );
```

Comment:
Nit: IMO it'd feel more natural to put this above the assessment questions route.

Reply:
done in dd2e51b894